engine/src/cmd/collide_map.h

```diff
@@ -195,6 +199,8 @@ class CollideMap : public CollideArray {
     bool CheckCollisions(Unit *un, const Collidable &updated); //will be handed off to a templated function
     bool CheckUnitCollisions(Unit *un,
             const Collidable &updated); //DANGER must be used on lists that are only populated with Units, not bolts
+
+    // TODO: Add virtual destructor?
```

Comment:
Yes, even if just empty and/or calling the parent

Reply:
Agreed. That is most likely what needs to happen, sooner or later. But can it wait for another PR?

Reply:
yes, we probably need to do a PR with adding it to everything we can find; though adding:
```c++
virtual ~CollideMap() { ~ListMutableSet<Collidable>() }
```
here would do well too.

Reply:
I don't think virtual destructors need to call each other. IIRC, they automatically get called in the proper sequence.

Making the destructor virtual, for every class that is intended to be inheritable, IS something that needs to happen. But I think in this case, I'll wait until the next PR.

Thanks for the review!

Reply:
Filed an issue for this. Resolving conversation for now.